client/components/input-chrono/style.scss

```diff
@@ -1,10 +1,10 @@
-.input-chrono__container {
+.input-chrono {
 	position: relative;
 	margin: 6px auto;
 
 	.gridicons-calendar {
 		color: var( --color-neutral-100 );
-		z-index: z-index( '.popover', '.input-chrono__container .gridicons-calendar' );
+		z-index: z-index( '.popover', '.input-chrono .gridicons-calendar' );
```

Comment:
I didn't find any instance of the `calendar` gridicon (or any other icon) inside the `InputChrono` component. It's just text.

Reply:
I wasn't quite sure how z-index works, so I thought I might retain this styling as well. I see that the gridicon is not used anywhere, so I have gone ahead and remove this entire styling on 6ee7d3a9ee8243a3b84c9c8435bbc4fcb39fb7b6.